src/FlexTable.jl

```diff
@@ -243,14 +243,37 @@ end
 # Private fields are never exposed since they can conflict with column names
 Base.propertynames(t::FlexTable, private::Bool=false) = columnnames(t)
 
+
+Base.vcat(t::FlexTable) = t
+
+function Base.vcat(t::FlexTable{<:NamedTuple{names}}, t2::FlexTable{<:NamedTuple{names}}) where {names}
+    return FlexTable(map(vcat, columns(t), columns(t2)))
+end
+
+function Base.vcat(t1::FlexTable{<:NamedTuple{names}}, t2::FlexTable{<:NamedTuple{names}}, ts::FlexTable{<:NamedTuple{names}}...) where {names}
```

Comment:
The signature below is a better template, which should allow us to cover the cases where there are mixtures of `Table`s and `FlexTable`s.